src/components/PulsatingCircle.tsx

```diff
@@ -0,0 +1,46 @@
+import { useCallback, useEffect, useState } from 'react';
+import styles from './PulsatingCircle.module.css';
+import { createPortal } from 'react-dom';
+
+interface PulsatingCircleProps {
+    anchor: Element;
+};
+
+export default function PulsatingCircle({ anchor }: PulsatingCircleProps) {
+    const calculatePos = useCallback(() => {
+        const box = anchor.getBoundingClientRect();
+        const viewport = document.documentElement;
+
+        const centerX = box.x + viewport.scrollLeft + box.width / 2;
+        const centerY = box.y + viewport.scrollTop + box.height / 2;
+        const size = Math.max(box.width, box.height) + 5;
+        
+        return [centerX, centerY, size];
+    }, [anchor]);
+
+    const [[x, y, size], setPos] = useState(calculatePos());
+
+    useEffect(() => {
+        setPos(calculatePos());
+    }, [anchor, calculatePos]);
+
+    useEffect(() => {
+        function handleResize() {
+            setPos(calculatePos());
+        }
+
```

Comment:
Ta bort onödig `useEffect` och kalla `handleResize` innan vi slänger till event listenern.
```suggestion
    useEffect(() => {
        function handleResize() {
            setPos(calculatePos());
        }

        handleResize();
```

Reply:
Smart, jag håller med! Har ändrat till det